### PR TITLE
print_all_ivy_arrays_on_dev

### PR DESCRIPTION
**Pull request by tamukagate**

### CODE REVIEW

**Comment by tamukagate** [comment]
Close #5498

**Comment by CheezBallzPi** [comment]
Thanks for contributing! Can you add the Formatting Checklist to your PR as shown in this guide? https://lets-unify.ai/ivy/contributing/4_open_tasks.html#formatting-checklist

**Comment by tamukagate** [comment]
add_reformatting_checklist_device

File: ivy/functional/backends/jax/device.py [hunk 1]
```diff
@@ -28,6 +28,15 @@ def _to_array(x):
 # ----#
 
 
+def print_all_ivy_arrays_on_dev(
```
Comment (CheezBallzPi) [inline ivy/functional/backends/jax/device.py:31]: Since all the work is being done outside the backends you can remove these.

**Review by CheezBallzPi — CHANGES_REQUESTED** [review]

**Comment by CheezBallzPi** [comment]
The failing tests have nothing to do with your code. You can safely ignore them.

**Review by CheezBallzPi — APPROVED** [review]
LGTM

**Pull request merged**